ContactUs.html

```diff
@@ -1,96 +1,40 @@
+<!DOCTYPE html>
 <html>
 
+<head>
+  <title> C - N Concise Networks</title>
+  <link rel="stylesheet" type="text/css" href="./assets/styles/HomePageStyle.css">
+  <script src="HomeScript.js"> </script>
+</head>
 
-<style>
-		* {
-    box-sizing: border-box;
-}
-
-body {
-    font-family: Arial, Helvetica, sans-serif;
-}
-
-/* Style the header */
-.header {
-	background-image: url("HeaderBackground.jpg");
-    background-repeat: no-repeat;
-    background-attachment: fixed;
-    padding: 30px;
-    text-align: center;
-    font-size: 35px;
-}
-
-/* Create three equal columns that floats next to each other */
-.column {
-    float: left;
-    width: 33.33%;
-    padding: 10px;
-	text-align: center;
-    height: 50px; /* Should be removed. Only for demonstration */
-}
-
-/* Clear floats after the columns */
-.row:after {
-    content: "";
-    display: table;
-    clear: both;
-}
-
-.column1 {
-	background-image: url("HomeBackground.jpg");
-}
-
-.column2 {
-	background-image: url("AboutUsBackground.jpg");
-}
-
-.column3 {
-	background-image: url("ContactUsBackground.jpg");
-}
+<body>
 
-.footer {
-	position: absolute;
-	right: 0;
-	bottom: 0;
-	left: 0;
-	padding: 1rem;
-	background-image: url("HeaderBackground.jpg");
-	text-align: center;
-}
+  <script></script>
 
-/* Responsive layout - makes the three columns stack on top of each other instead of next to each other */
-@media (max-width: 600px) {
-    .column {
-        width: 100%;
-    }
-}
-</style>
-	<head>
-	 <link rel="stylesheet" type="text/css" href="HomePageStyle.css"> 
-		<title> C - N Concise Networks</title>
-</head>
-	<body>
-	
-		<script "></script>
-		
-	<div class="header">
-  <h2>C - N Concise Networks</h2>
-</div>
+  <div class="header ">
+    <h2>C - N Concise Networks</h2>
+  </div>
 
-<!--Column Links-->
+  <!--Column Links-->
 
-<div class="row">
-  <div class="column column1"><a href="file:///C:\Users\PC\Desktop\WebDev\HomePage.html">Home</a></div>
-  <div class="column column2"><a href="file:///C:\Users\PC\Desktop\WebDev\AboutUs.html">About Us</a></div>
-  <div class="column column3"><a href="file:///C:\Users\PC\Desktop\WebDev\ContactUs.html">Contact Us</a></div>
-</div>
+  <div class="row ">
+    <div class="column column1 ">
+      <a href="HomePage.html ">Home</a>

```

Comment:
I've changed the links to be relative to itself, rather than relative to the harddrive. This is a much better approach for creating links.